frontend/src/components/scan/ScanForm.jsx

```diff
@@ -239,8 +249,8 @@ export default function ScanForm() {
         connectors: values.connectors.map((x) => x.value),
       };
       try {
-        const jobId = await createJob(formValues);
-        setTimeout(() => navigate(`/jobs/${jobId}`), 1000);
+        await createJob(formValues);
+        setTimeout(() => navigate(`/jobs/`), 1000);
```

Comment:
if the request would be a single request, the behaviour should be the same as before. This is correct only